### PR TITLE
mac: fixed framework codesign

### PR DESCRIPTION
**Pull request by pingwinator**
HI.
I updated framework sign options. 
We have tried to build it for https://github.com/bvschaik/julius/issues/579 and had a codesign problem.

### CODE REVIEW

File: Xcode/SDL_mixer.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -521,7 +521,7 @@
 		0867D690FE84028FC02AAC07 /* Project object */ = {
 			isa = PBXProject;
 			attributes = {
-				LastUpgradeCheck = 0900;
+				LastUpgradeCheck = 1250;
```
Comment (sezero) [inline Xcode/SDL_mixer.xcodeproj/project.pbxproj:524]: This part isn't necessary, yes?

**Comment by sezero** [comment]
Tweaked the commit message a little and omitted the LastUpgradeCheck part,
applied as https://github.com/libsdl-org/SDL_mixer/commit/5c54e5449a4ba5bbfdad4d9f034dc93e182c6c65

Thanks.

File: Xcode/SDL_mixer.xcodeproj/project.pbxproj (hunk 1 above)
Comment (pingwinator) [reply Xcode/SDL_mixer.xcodeproj/project.pbxproj:524]: you could skip it. thx

**Comment by sezero** [comment]
I'm guessing that the arm64 builds of the music frameworks work and we can close
https://github.com/libsdl-org/SDL_mixer/issues/324 and https://github.com/libsdl-org/SDL_mixer/issues/291 ?

**Comment by pingwinator** [comment]
> I'm guessing that the arm64 builds of the music frameworks work and we can close
> #324 and #291 ?

#291 was open from the thread bvschaik/julius#579 So you could close it
Can you release a new version and publish new official binaries, please?

**Comment by slouken** [comment]
I can test this on M1 macOS, and it's on my TODO list.

**Pull request closed without merging**